portfolio/src/main/webapp/script.js

```diff
@@ -117,19 +127,60 @@ function initMap() {
     content: contentString
   });
 
-  marker.addListener('click', function() {
-    if (marker.getAnimation() === google.maps.Animation.BOUNCE)  
-      marker.setAnimation(null);
-    else {
-      infowindow.open(map, marker);
-      marker.setAnimation(google.maps.Animation.BOUNCE);
+  marker.addListener("click", function() {
+    infowindow.open(map, marker);
+    marker.setAnimation(google.maps.Animation.BOUNCE);
       setTimeout(function() { marker.setAnimation(null); }, 600);
-    }
   });
+}
+
+var markers = [];
+var marker;
+
+function addPin(comment, time, i) {
```

Comment:
I think `i` should have a more descriptive name in the function declaration, or maybe better yet left out entirely. You can just do the computation for the delay at the call-site.

Reply:
I somehow skipped over just this comment, addressing it in new PR along with code cleanup.